client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -9,7 +9,7 @@
 	.design-selector__header {
 		display: flex;
 		align-items: center;
-		margin-top: 53px;
+		margin-top: 60px;
```

Comment:
why `60px` and not `64px`? maybe we can also work with the height of the heading itself.

Reply:
This is because of how type is displayed, we naturally get more padding at the bottom than on the top. I'm just compensating for that.